packages/mobile_app/lib/main.dart

```diff
@@ -1,45 +1,52 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
-import 'package:local_database/local_database.dart';
+import 'package:prophecies_flutter/prophecies_flutter.dart';
+import 'package:auth_flutter/auth_flutter.dart';
+import 'package:users_flutter/users_flutter.dart';
+import 'package:provider/provider.dart';
 
 import 'blocs/blocs.dart';
 import 'blocs/simple_bloc_delegate.dart';
-import 'theme/app_theme.dart';
-import 'views/home_screen.dart';
+import 'routes.dart';
 
 void main() {
   BlocSupervisor.delegate = SimpleBlocDelegate();
   runApp(ProphetApp());
 }
 
 class ProphetApp extends StatelessWidget {
+  final usersStorage = UsersStorageJson();
+
   @override
-  Widget build(BuildContext context) => MultiBlocProvider(
+  Widget build(BuildContext context) => MultiProvider(
         providers: [
+          Provider<UsersStorageJson>(create: (context) => usersStorage),
+        ],
+        child: MultiBlocProvider(providers: [
           BlocProvider<AuthenticationBloc>(
-              create: (context) => AuthenticationBloc(
-                    userRepository: UserRepository.createDefault(),
-                  )..add(AppStarted())),
+            create: (context) =>
+                AuthenticationBloc(auth: AuthFlutter(storage: usersStorage))
+                  ..add(AppStarted()),
+          ),
           BlocProvider<PropheciesBloc>(
             create: (context) => PropheciesBloc(
-              repository: PropheciesRepository(context),
+              repository: PropheciesFlutter(context),
```

Comment:
Worth renaming to `PropheciesRepositoryFlutter`?

Reply:
Yeah, I will do reanming it all and others to repositories.